src/main/kotlin/pt/up/fe/ni/website/backend/model/Event.kt

```diff
@@ -19,7 +22,24 @@ class Event(
     val description: String,
 
     @JsonProperty(required = true)
-    val date: Date,
+    val startDate: Date,
+
+    val endDate: Date?,
+
+    @field:NullOrNotBlank
+    @field:URL
+    val url: String?,
+
+    @field:Size(min = Constants.Location.minSize, max = Constants.Location.maxSize)
+    val location: String?,
+
+    @field:Size(min = Constants.Category.minSize, max = Constants.Category.maxSize)
+    val category: String?,
```

Comment:
I'm not against it but what is your idea behind the category field? What values would it have on the website?
I'm asking so we don't merge unnecessary code

Reply:
I get it, it makes sense. Do you think it's worth doing that right now? We could maybe create an issue for later